web/elm/src/Build/Build.elm

```diff
@@ -686,6 +696,19 @@ tooltip _ { hovered } =
                 , arrow = Just { size = 5, color = Colors.tooltipBackground }
                 }
 
+        HoverState.Tooltip (StepHeaderImageFetching _) _ ->
+            Just
+                { body =
+                    Html.div
+                        Styles.changedStepTooltip
+                        [ Html.text "initialization" ]
```

Comment:
Question: is volume streaming considered part of initialization? And if so, might it be confusing to call the image fetching panel initialization? If the initialization takes 1 minute, but the `check` and `get` steps only take 10 seconds, I can see people being confused where the other 50 seconds wound up

Reply:
Hmm yeah true. I was thinking we might add more initialization information there in the future, but I can change it to 'image fetching' or something so it's accurate until then.

It might be a good place to move the selected worker name, which was added in #5846 and is currently printed to the logs as a stopgap. I hacked it together for a sec but couldn't figure out a good UI quickly so I bailed on it for now.

Reply:
I like the idea of adding more initialization info in the future - would be cool to add e.g. progress bars tracking artifact streaming. Before I understood what Concourse was doing a bit better, I was confused why a step could be spinning for a while with no logs being emitted, so revealing what's going on behind the scenes in the UI is a big plus